readme.md

```diff
@@ -19,6 +19,12 @@ If the above does not work, in enterprise environments, try:
 libraryDependencies += "org.scalameta" % "sbt-scalafmt_2.12_1.0" % PLUGIN_VERSION
 ```
 
+#### JDK compatibility
```

Comment:
should it be under `installation`? this file doesn't have any other `####` headings, everything is `###`.

Reply:
Yes, that was my goal (to have this as a sub-section of `installation` section). If you prefer it to be a "h3" section, I will happily change this.